L1Trigger/TrackFindingTracklet/interface/Cabling.h

```diff
@@ -0,0 +1,160 @@
+// This class holds a list of stubs that are in a given layer and DCT region
+#ifndef CABLING_H
+#define CABLING_H
+
+#include "L1TStub.h"
+#include "Stub.h"
+#include "DTCLink.h"
+#include "DTC.h"
+#include "Util.h"
+
+using namespace std;
```

Comment:
Please don't `using namespace std` in a header file.

Reply:
fixed by migrating to h/cc files